test/e2e/broker_trigger_test.go

```diff
@@ -56,28 +56,17 @@ type eventReceiver struct {
 // and sends different events to the broker's address. Finally, it verifies that only
 // the appropriate events are routed to the subscribers.
 func TestDefaultBrokerWithManyTriggers(t *testing.T) {
-	clients, cleaner := Setup(t, t.Logf)
-
-	// Verify namespace exists.
-	ns, cleanupNS := CreateNamespaceIfNeeded(t, clients, t.Logf)
```

Comment:
Just to be sure I understand, this means that the namespace _must_ be created before the test starts? Either via the normal shell scripts or manually if a user is directly calling `go test`?

Reply:
I've decided to create a new namespace for each test case we are running, and delete it after the test is finished. Please see the latest comment below.